src/main/java/com/amazonaws/encryptionsdk/kmsv2/AwsKmsMrkAwareMasterKeyProvider.java

```diff
@@ -100,8 +95,7 @@ public Builder withDefaultRegion(String defaultRegion) {
      *
      * @param discoveryMrkRegion The region to contact to attempt to decrypt multi-region keys.
      */
-    public Builder withDiscoveryMrkRegion(
-        String discoveryMrkRegion) {
+    public Builder discoveryMrkRegion(Region discoveryMrkRegion) {
```

Comment:
Do we need to support accepting a string as well? https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html

Reply:
For v2, they've pretty strongly moved away from strings in place of the typed `Region`. If for some reason a user wants to use some custom region, they can use `Region.of("custom-region-1")`